Description: Add missing OUTPUT references to CODE sections
 Two functions are using a RETVAL function but failed to include a
 corresponding OUTPUT section to actually produce the values. I've
 included such sections in this patch.
Author: Peter J. Mello <admin@petermello.net>
Origin: Asbru Project Team <https://github.com/asbru-cm/packages/blob/master/libgnome2-gconf-perl/debian/patches/retval-but-no-output.patch>
Forwarded: no
Last-Update: 2019-05-27

--- a/xs/GConfEngine.xs
+++ b/xs/GConfEngine.xs
@@ -188,6 +188,8 @@ gconf_engine_get_without_default (engine
     	RETVAL = gconf_engine_get_without_default (engine, key, &err);
 	if (err)
 		gperl_croak_gerror (NULL, err);
+    OUTPUT:
+        RETVAL
 
 =for apidoc
 
@@ -211,6 +213,8 @@ gconf_engine_get_with_locale (engine, ke
     	RETVAL = gconf_engine_get_with_locale (engine, key, locale, &err);
 	if (err)
 		gperl_croak_gerror (NULL, err);
+    OUTPUT:
+        RETVAL
 
 =for apidoc
 
